regolith/helpers/l_abstracthelper.py

```diff
@@ -55,11 +55,6 @@ def construct_global_ctx(self):
         if "groups" in self.needed_dbs:
             rc.pi_id = get_pi_id(rc)
         rc.coll = f"{TARGET_COLL}"
-        try:
```

Comment:
I guess you are handling this somewhere else in the helper, but I don't see it.  How do you handle the choice of database to make the update to?

Reply:
@sbillinge I took the attempt at getting the database keyword out of these helpers because it is not present in the first place. 

In the updaters and adders you must specify which database you want to edit, but if you look a the subparse method for these listers, there was no database keyword requested. That is why they all had exception handlers (they all raise exceptions because the key is not present). This was confusing (made me think that these helpers did specify which database to list from) and unnecessary, so I removed it.

Reply:
nice, thanks!